benchmark/fft/fft_config.cc

```diff
@@ -61,11 +61,13 @@ bool FFTConfig::Parse(int argc, char** argv) {
   parser.AddFlag<base::BoolFlag>(&check_results_)
       .set_long_name("--check_results")
       .set_help("Whether checks results generated by each fft runner.");
-  parser.AddFlag<base::Flag<std::vector<Vendor>>>(&vendors_)
-      .set_long_name("--vendor")
-      .set_help(
-          "Vendors to be benchmarked with. (supported vendors: arkworks, "
-          "bellman, halo2)");
+  if (options.include_vendors) {
```

Comment:
What is the motivation behind making `include_vendors` flag optional?

Reply:
This is because the `fft_benchmark_gpu` does not have a vendor flag, whereas the `fft_benchmark` does have a vendor flag.